src/Zynga/Framework/Cache/V2/Driver/Memcache.hh

```diff
@@ -79,6 +82,39 @@ class Memcache extends DriverBase {
 
   }
 
+  public function directSet(
```

Comment:
You will run into few changes into master that i did. I made an interface for Memcache and InMemory driver. Can you add directSet and directGet to the interface and also implement that for the InMemory driver.

Reply:
Absolutely.